### ISSUE TITLE
Cookiebot

### ISSUE THREAD

**Issue by ShankarSumanth** [junaid33/opensource.builders#201]
A GDPR based cookie and third party cookie  implementation handling for websites.

**Comment by jcklpe** [comment]
@ShankarSumanth  how does Cookie Consent fit as a comparison? https://www.osano.com/cookieconsent

**Comment by ShankarSumanth** [comment]
@jcklpe Yes this fits. The free tier is enough for startup businesses. Thanks

**Comment by jcklpe** [comment]
@ShankarSumanth  actually we leave these issues open so that we can use them as reference for others.

**Comment by ShankarSumanth** [comment]
ah ok.. Did not know that :)
